AUTHORS

```diff
@@ -731,6 +732,7 @@ Feroz Salam <feroz.salam@sourcegraph.com>
 Ferran Rodenas <frodenas@gmail.com>
 Filipe Brandenburger <filbranden@google.com>
 Filipe Oliveira <contato@fmoliveira.com.br>
+Filipe Pina <636320+fopina@users.noreply.github.com>
```

Comment:
Looks like this one can use an entry in `.mailmap` for the e-mail address;

```bash
git log --author=636320+fopina@users.noreply.github.com
commit ef681124ca61f52e72d1e1c4fc589373d2a4e08e
Author: Filipe Pina <636320+fopina@users.noreply.github.com>
Date:   Thu Feb 29 23:23:19 2024 +0000

    fix typo in error message

    Signed-off-by: Filipe Pina <hzlu1ot0@duck.com>
```

Reply:
Good catch; missed this one!